backfill/2024-04-26-active_users_aggregates/fenix_query.sql

```diff
@@ -1,4 +1,20 @@
-WITH baseline AS (
+WITH distribution_id AS
+(
+  SELECT
+    client_info.client_id,
+    ARRAY_AGG(
+      metrics.string.metrics_distribution_id IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS distribution_id
+  FROM
+    `moz-fx-data-shared-prod.fenix.metrics`
+  WHERE
+    DATE(submission_timestamp) BETWEEN @submission_date AND DATE_ADD(@submission_date, INTERVAL 1 DAY)
```

Comment:
This query only runs once per day, right? Is there a big downside to broadening this window to something like:

```sql
DATE(submission_timestamp) BETWEEN DATE_SUB(@submission_date, INTERVAL 7 DAY) AND DATE_ADD(@submission_date, INTERVAL 7 DAY)
```

Reply:
Everything looks good in the PR; @bani [just brought up an interesting data point to consider](https://mozilla.slack.com/archives/C06LZ00TARL/p1716906155459829?thread_ts=1716412956.224719&cid=C06LZ00TARL) so while I don't want to keep delaying this backfill I think it's worth being certain that using the metrics ping is the best path forward.